trax/rl/training.py

```diff
@@ -185,7 +185,11 @@ def policy(self, trajectory):
     """Chooses an action to play after a trajectory."""
     model = self._policy_eval_model
     model.weights = self._policy_trainer.model_weights
-    pred = model(trajectory.last_observation[None, ...], n_accelerators=1)[0]
+    trajectory_np = trajectory.to_np(timestep_to_np=self.task.timestep_to_np)
+    # Add batch dimension to trajectory_np and run the model.
+    pred = model(trajectory_np.observations[None, ...], n_accelerators=1)
```

Comment:
Looks like it will run model on all past observations from the trajectory, no only on the latest one, doesn't it?